cmd/upgrade/upgrade.go

```
@@ -0,0 +1,355 @@
+		tmpFile.Close()
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		tmpFile.Close()
+		return fmt.Errorf("download returned status %d", resp.StatusCode)
+	}
+
+	_, err = io.Copy(tmpFile, resp.Body)
+	tmpFile.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `tmpFile.Close` is not checked

(errcheck)